### PR TITLE
V2 enable ubuntu

### PR DESCRIPTION
**Pull request by vrothberg**
Meant to replace https://github.com/containers/libpod/pull/6418.

### CODE REVIEW

**Comment by vrothberg** [comment]
Don't merge. Added a bad commit but want to let it run to test.

**Comment by rhatdan** [comment]
@vrothberg Could you squash your commits.  "m" is a very informative commit message, but I have a `Q` about it.  :^)

**Comment by vrothberg** [comment]
> @vrothberg Could you squash your commits. "m" is a very informative commit message, but I have a `Q` about it. :^)

he, I noticed too but want to wait and see if that turns tests green.

File: hack/install_bats.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+set -e
+
+die() { echo "${1:-No error message given} (from $(basename $0))"; exit 1; }
+
+buildDir=$(mktemp -d)
+git clone https://github.com/bats-core/bats-core $buildDir
+
+pushd $buildDir
+echo `pwd`
```
Comment (edsantiago) [inline hack/install_bats.sh:11]: nit (minor, no need to fix): it's cleaner to just say `pwd`

Comment (vrothberg) [reply hack/install_bats.sh]: Thanks, fixed now

File: test/e2e/checkpoint_test.go [hunk 2]
```diff
@@ -232,6 +232,7 @@ var _ = Describe("Podman checkpoint", func() {
 	})
 
 	It("podman checkpoint container with established tcp connections", func() {
+		Skip("fails on ubuntu")
```
Comment (mheon) [inline test/e2e/checkpoint_test.go:235]: Why is this a plain Skip? Shouldn't this be a `SkipIfNotFedora()` too?

Comment (vrothberg) [reply test/e2e/checkpoint_test.go:235]: Inherited from #6418 but a good catch! Will update with the next push.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests and @mheon's comment is addressed

**Comment by edsantiago** [comment]
The BATS failure is an Ubuntu bug: they forked their own BATS and broke it. I've filed an [Ubuntu bug](https://bugs.launchpad.net/ubuntu/+source/bats/+bug/1882542)

**Comment by vrothberg** [comment]
Can someone drop a 'lgtm hold'?

**Comment by edsantiago** [comment]
bindings failed. I'm assuming it's a flake. Restarted.

**Pull request merged**